nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -688,6 +689,35 @@ define([
             });
         }
 
+        /**
+         * Toggle the display of the specified tab
+         * If the tab is already being displayed, this will hide it
+         *
+         * @param {string} tab id of the tab to display
+         */
+        function toggleTab(tab) {
```

Comment:
differentiate between toggleTab (can collapse a currently-displayed tab) and switchToTab (won't collapse a current tab)